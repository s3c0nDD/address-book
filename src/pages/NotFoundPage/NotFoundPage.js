import React from 'react';
import { makeStyles } from '@material-ui/core/styles';

import Grid from '@material-ui/core/Grid';
import Typography from '@material-ui/core/Typography';

const useStyles = makeStyles({
  root: {
    height: 200,
  },
  text: {
    margin: '0 30px'
  }
});

/**
 * Not found page component
 */
const NotFoundPage = () => {
  const classes = useStyles();

  return (
    <Grid
      className={classes.root}
      container
      justify="center"
      alignItems="center"
    >
      <Grid item>
        <Typography
          className={classes.text}
          variant="h4"
          color="textSecondary"
        >
          404 Page Not Found :(
        </Typography>
      </Grid>
    </Grid>
  );
};

export default NotFoundPage;
