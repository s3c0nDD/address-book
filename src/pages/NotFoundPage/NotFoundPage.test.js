import React from 'react';
import { createShallow } from '@material-ui/core/test-utils';

import NotFoundPage from './NotFoundPage'

describe('NotFoundPage', () => {
  let shallow;
  let wrapper;

  beforeAll(() => {
    shallow = createShallow();
  });

  beforeEach(() => {
    wrapper = shallow(
      <NotFoundPage />
    );
  });

  afterEach(() => {
  });

  it('shallowly renders and matches snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});