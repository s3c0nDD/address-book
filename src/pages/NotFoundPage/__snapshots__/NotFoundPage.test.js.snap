// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage shallowly renders and matches snapshot 1`] = `
<WithStyles(ForwardRef(Grid))
  alignItems="center"
  className="makeStyles-root-1"
  container={true}
  justify="center"
>
  <WithStyles(ForwardRef(Grid))
    item={true}
  >
    <WithStyles(ForwardRef(Typography))
      className="makeStyles-text-2"
      color="textSecondary"
      variant="h4"
    >
      404 Page Not Found :(
    </WithStyles(ForwardRef(Typography))>
  </WithStyles(ForwardRef(Grid))>
</WithStyles(ForwardRef(Grid))>
`;
